types/activex-excel/index.d.ts

```diff
@@ -2,13 +2,14 @@
 // Project: https://msdn.microsoft.com/en-us/library/fp179694.aspx
 // Definitions by: Zev Spitz <https://github.com/zspitz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.6
 
 /// <reference types="activex-office" />
 /// <reference types="activex-vbide" />
 /// <reference types="activex-stdole" />
 
 declare namespace Excel {
+    // tslint:disable-next-line:no-const-enum
```

Comment:
I think that the most appropriate change would be to change these to regular enums.

Reply:
@DanielRosenwasser These enums don't exist at runtime; in other words the following Javascript will fail:

    var x = Excel.Constants.xl3DBar;

But they are still useful as [documented constant values](https://github.com/Microsoft/TypeScript/issues/20703#issuecomment-361914358).

I understand this is a problem when `--isolatedModules`, but my understanding was that the final fix will be to [error on files consuming enums this way when `--isolatedModules` has been set.](https://github.com/Microsoft/TypeScript/issues/20703#issuecomment-361434795).

Reply:
@DanielRosenwasser Any further word on this?